roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -94,6 +100,19 @@ def subclasses(cls):
             for subclass in subclass.subclasses():
                 yield subclass
 
+    def register_failure(self, error):
```

Comment:
Why is this part of the `OpenShiftCheck` class? When can a check have more than one failure?

Reply:
Frequently. For instance, the disk availability check previously would fail on the first file system it found that was too full; now it will report multiple problem paths. The EFK checks often check multiple things and put together a list of failures, and this is actually a bit messy when they're coming from different methods. Easier if you can just record it where it's found and let it be summarized at the end. Of course you can also raise an exception if the check can't proceed after a failure.